src/fr/disabilities/index.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Types d'invalidité les plus courants
+layout: layouts/base.njk
+description: Voici quelques-uns des types de handicaps les plus courants sur le lieu de travail, et les considérations d'accessibilité qui s'y rapportent.
+---
+
+<div class="row wb-eqht">
```

Comment:
Suggestion:
Use equal height in pure CSS and leverage the service and information component